hugegraph-core/src/main/java/com/baidu/hugegraph/schema/builder/EdgeLabelBuilder.java

```diff
@@ -141,6 +141,72 @@ public EdgeLabel create() {
         });
     }
 
+    private boolean hasSameProperties(EdgeLabel existedEdgeLabel) {
```

Comment:
prefer to move this method to class EdgeLabel, and process parent fields by each super class like SchemaLabel/SchemaElement

Reply:
If move hasSameProperties to EdgeLabel, and parent class, it will reduce some duplicate check. 
SchemaLabel has the following fields used by VertexLabel and EdgeLabel.
```
private final Set<Id> properties;
    private final Set<Id> nullableKeys;
    private final Set<Id> indexLabels;
    private boolean enableLabelIndex;
```
But the parameter should be Builder, and in should has proper method to retrieve correspond information.  There is not common parent Builder with these information.